src/modules/redesign/types/components/vehicle.d.ts

```diff
@@ -210,7 +209,7 @@ export type RedesignVehicleComponent = RedesignComponent<
         sortedItems: ItemChooser<'item'>[];
         shownItems: ItemChooser<'item'>[];
 
-        participatedMissions: number[];
+        participatedMissions: string[];
```

Comment:
String for "123/a" etc?

Reply:
nopes. See comment for the `Object.keys` and `parseInt` thing.